tox.ini

```diff
@@ -48,10 +48,12 @@ commands =
     py.test --create-db -n 2 -m 'not es_tests' -v src/olympia/ --ignore src/olympia/addons/ --ignore src/olympia/devhub/ --ignore src/olympia/editors/ --ignore src/olympia/amo/ --ignore src/olympia/users/ {posargs}
 
 [testenv:ui-tests]
-passenv = DISPLAY
-install_command = pip install {packages}
-deps = -rtests/ui/requirements.txt
-commands = pytest --verify-base-url --driver Firefox tests/ui {posargs}
+recreate = True
+passenv = DISPLAY PYTEST_ADDOPTS PYTEST_BASE_URL ELASTICSEARCH_LOCATION
+commands =
+    make update_deps
+    pip install --exists-action=w -r tests/ui/requirements.txt -r requirements/dev.txt
```

Comment:
This was @EnTeQuAk. He suggested this as it would follow the rest of the tox file. I'd rather ```install_command``` too.